### PR TITLE
add HW_6

### PR DESCRIPTION
**Pull request by andreyknyshov**
Прошу проверить ДЗ №6

### CODE REVIEW

File: HW_5/hw_5.2.1.js [hunk 1]
```diff
@@ -17,30 +17,25 @@
...
-      break;
-    case 3: 
-      console.log(bytes.toFixed(1),"Gb");
-      break;
-  }
-  
-  
-  
-   
+while (bytes >= 1024) {
```
Comment (anatoly-karpovich) [inline HW_5/hw_5.2.1.js:23]: no check for currentUnit > 3

File: HW_6/hw_6.1.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  const employeeArray = ["Иван", "Елена", "Максим", "Владислав", "Андрей"];
+  const paymentArray = [50000, 56000, 68000, 45000, 70000];
+  const index = employeeArray.indexOf(employeeName); // находим индекс сотрудника в массиве имен сотрудников
+  if (index !== -1) {
+    // если индекс валидный и сотрудник найден - формируем выходной массив
+    const resultArray = [];
+    resultArray.push(employeeName, paymentArray[index]);
+    return resultArray;
+  } else {
+    return null;
```
Comment (anatoly-karpovich) [inline HW_6/hw_6.1.js]:
```suggestion
    return resultArray.length ? resultArray : null
```

File: HW_6/hw_6.2.1.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  "Diablo",
+  "4 cheeses",
+  "hawai",
+];
+
+function getUniquePizza(myPizzaArray) {
+  const resultArray = [];
+  for (let indexMy = 0; indexMy < myPizzaArray.length; indexMy++) {
+    let isTheSame = false;
+    for (let indexComp = 0; indexComp < competitorPizzas.length; indexComp++) {
```
Comment (anatoly-karpovich) [inline HW_6/hw_6.2.1.js:19]: use .includes() instead of nested loop

File: HW_6/hw_6.2.3.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  const uniqueNumbers = []; // Массив для уникальных чисел
+  for (let i = 0; i < numbers.length; i++) {
+    let isUniqueNumber = true;
+    // Проверка - было ли такое число уже в uniqueNumbers
+    for (let j = 0; j < uniqueNumbers.length; j++) {
+      if (numbers[i] === uniqueNumbers[j]) {
+        isUniqueNumber = false;
+        break;
+      }
+    }
```
Comment (anatoly-karpovich) [inline HW_6/hw_6.2.3.js:14]:
```suggestion
for(const num of number) {
  if(!uniqueNumbers.includes(num)) {
    uniqueNumbers.push(num)
  }
}
```

**Review by anatoly-karpovich — CHANGES_REQUESTED** [review]

File: HW_6/hw_6.2.4.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
+//4. Написать функцию, которая принимает на вход слово и проверяет, является ли это слово палиндромом
+
+function Palindrome(word) {
```
Comment (VladimirLQA) [inline HW_6/hw_6.2.4.js]:
```suggestion
function palindrome(word) {
```

Function names start in lower case, upper case is used for `constructor functions`

**Review by VladimirLQA — APPROVED** [review]

**Comment by andreyknyshov** [comment]
changed according to the mentors' comments

File: HW_6/hw_6.1.js [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+  const employeeArray = ["Иван", "Елена", "Максим", "Владислав", "Андрей"];
+  const paymentArray = [50000, 56000, 68000, 45000, 70000];
+  const index = employeeArray.indexOf(employeeName); // находим индекс сотрудника в массиве имен сотрудников
+  if (index !== -1) {
+    // если индекс валидный и сотрудник найден - формируем выходной массив
+    const resultArray = [];
+    resultArray.push(employeeName, paymentArray[index]);
+    return resultArray.length ? resultArray : null;
+  }
+}
```
Comment (anatoly-karpovich) [inline HW_6/hw_6.1.js:20]:
```suggestion
 return index === -1 ? null : [employeeName, paymentArray[index]];
}
```

**Review by anatoly-karpovich — APPROVED** [review]

**Pull request merged**